README.md

```diff
@@ -1,39 +1,118 @@
-# 3box-pinbot
+# 3box-ghost-pinbot
 
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-This package contains a CLI and a Dockerfile for spinning up your pinbot that resides in a Ghost Thread.
+This package contains information for spinning up your Ghost Pinbot server used for attaching/detaching to Ghost threads.
 
 ## Tabel of Contents
-- [Install](#install)
-- [Usage](#usage)
+- [Start](#start)
+- [Swarm](#swarm)
+- [APIs](#APIs)
 - [Contributing](#contributing)
 - [License](#license)
 
-## Install
+## Start
+
+There are various ways of how to start Ghost Pinbot server. There is a `Dockerfile` which can be used to build the image and deploy it on your infrastructure.
+
+The `docker-compose.yml` can be a good start to bootstrap Ghost Pinbot server:
+
 ```
-$ npm install -g 3box-pinbot
+$ docker-compose up
 ```
 
-## Usage
-Use the CLI to join a room locally from the console
-```shh
-# join with pinbot command and supply a room name
-pinbot 3box.ghost.<space-name>.<room-name>
+If you plan to deploy the Ghost Pinbot server on AWS there are Terraform scripts located in `deploy/aws-terraform-example`. 
+
+In order to setup secure WebSocket for `swarm` direct connect over `https` that's automatically implemented in the Terraform scripts provided. The only
+input that needs to be provided is the certificate arn from AWS.
+
+If you want to setup secure WebSocket for `swarm` without using AWS and the scripts provided, feel free to set it up using `nginx` or 
+other software to create a reverse proxy.
+
+## Swarm
+
+Upon successful start and deployment, the MultiAddress of a Ghost Pinbot node can be passed as an option `ghostPinbot` to `3Box` on create.
```

Comment:
Maybe add a sentence of how you can get the multiaddress of the pinbot.

Reply:
Makes sense 👍  I added a couple of utility routes